sycl/doc/UsersManual.md

```diff
@@ -410,6 +410,11 @@ and not recommended to use in production environment.
     Also, some of Intel GPUs or GPU run-time/drivers may support only
     "stateless" memory accesses.
 
+**`-fsycl-disable-fine-grained-bitfield-accesses`**
+
+    Disables the addition of `-ffine-grained-bitfeld-accesses` option for
+    SPIR based targets.
```

Comment:
I know you didn't ask for a review yet - but the description and help information probably shouldn't be mentioning the CC1 option but instead provides information about what the option does.

Reply:
Do we need a separate option for that? Shouldn't we unconditionally pass `-ffine-grained-bitfeld-accesses` if compile for the SPIR target w/o adding driver options?

Reply:
Based on discussions with @bader, following changes were made:
1. Removed the new option
2. Added two tests - sycl test to check code generation and driver test to check option inclusion

Thanks